<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDialog</class>
 <widget class="QDialog" name="InfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>227</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>227</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Folder Stats</string>
  </property>
  <widget class="QLabel" name="ttlImgLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ttFileSizeLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="uniqueDimLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="smallResLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="highResLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>201</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="ttlCountUILabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total # Images:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ttlFileSizeUILabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total File Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="highResUILabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Highest Resolution</string>
    </property>
   </widget>
   <widget class="QLabel" name="smallResUILabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Smallest Resolution:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ttlUniqueUILabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Unique Dimensions:</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>341</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>341</width>
      <height>32</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
